<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewsPage</class>
 <widget class="QWidget" name="NewsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(217, 217, 217);</string>
  </property>
  <layout class="QVBoxLayout" name="rootLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="contentStackedWidget">
     <widget class="QWidget" name="contentLoadingPage">
      <layout class="QVBoxLayout" name="contentLoadingPageLayout">
       <item>
        <widget class="QProgressBar" name="contentProgressBar">
         <property name="maximum">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="contentResultsPage">
      <layout class="QVBoxLayout" name="contentResultsPageLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="contentResultsScrollArea">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="contentResultsScrollAreaContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-146</y>
            <width>1092</width>
            <height>734</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="contentResultsScrollAreaContentsLayout">
           <property name="rightMargin">
            <number>24</number>
           </property>
           <property name="horizontalSpacing">
            <number>60</number>
           </property>
           <property name="verticalSpacing">
            <number>36</number>
           </property>
           <item row="5" column="0">
            <layout class="QHBoxLayout" name="navigationLayout">
             <property name="spacing">
              <number>0</number>
             </property>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="publishedLabel">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: black;</string>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="textFormat">
              <enum>Qt::TextFormat::PlainText</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="titleLabel">
             <property name="font">
              <font>
               <pointsize>25</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: black;</string>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
            <widget class="QPushButton" name="seeCommentsButton">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">background: linear-gradient(180deg,#e6e6e6,#e1e1e1);</string>
             </property>
             <property name="text">
              <string>See comments (%1)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="coverLabel">
             <property name="minimumSize">
              <size>
               <width>775</width>
               <height>200</height>
              </size>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="textFormat">
              <enum>Qt::TextFormat::PlainText</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QWebEngineView" name="descriptionWebEngineView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <property name="url">
              <url>
               <string>about:blank</string>
              </url>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2" rowspan="5">
            <layout class="QVBoxLayout" name="headlinesLayout">
             <property name="spacing">
              <number>16</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="headlinesTitleLabel">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>NEWS HEADLINES</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QStackedWidget" name="headlinesStackedWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>225</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>225</width>
                 <height>16777215</height>
                </size>
               </property>
               <widget class="QWidget" name="headlinesLoadingPage">
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QProgressBar" name="headlinesProgressBar">
                   <property name="maximum">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="headlinesResultsPage">
                <layout class="QVBoxLayout" name="headlinesResultsPageLayout">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
